examples/calculator/run_calculator.py

```diff
@@ -0,0 +1,72 @@
+import asyncio
+
+from yagna_requests import Session
+
+executor_cfg = {'budget': 10, 'subnet_tag': 'devnet-beta.2'}
+session = Session(executor_cfg)
+
+
+@session.startup('http://calc', '040e5b765dcf008d037d5b840cf8a9678641b0ddd3b4fe3226591a11', 3)
```

Comment:
`3` without param name looks very mysterious. It should be a named parameter (at least in the example).

Reply:
c417e62